vkext/vkext.cpp

```diff
@@ -532,6 +571,47 @@ int win_to_utf8(const char *s, int len) {
   return cur_buff_len;
 }
 
+static int json_escape(const char *string, int len) {
```

Comment:
What do you think about extracting this logic as well as `write_buff_char_*` and related buffers, somewhere to common and then use the single code in kphp and vkext?